lib/cmd/fh3/admin/mbaas/.delete.js

```diff
@@ -0,0 +1,15 @@
+// 2016-04-07: This command was temporarily disabled due to decision of not supporting mbaas deletion
```

Comment:
I thought it was a nice way to "disable" the command, do you have some recommendation for a better way?